compare_vcf.py

```diff
@@ -191,9 +191,9 @@ def run(self):
         else:
             if self.log_to_file:
                 with utils.versatile_open(self.log_to_file, 'a') as logout:
-                    utils.run_shell_command(cmd, sys.stdout, logout)
+                    utils.run_shell_command(cmd, sys.stderr, logout)
             else:
-                utils.run_shell_command(cmd, sys.stdout, sys.stderr)
+                utils.run_shell_command(cmd, sys.stderr, sys.stderr)
```

Comment:
@yunfeiguo why did you change stdout to stderr here?

Reply:
so both rtg and VarSim will print to stdout, but what rtg prints is only intermediate results. replacing `stdout` by `stderr` will make it possible to save the important stats by simple redirection `>`.